modules/carto/src/basemap.ts

```diff
@@ -1,4 +1,91 @@
-const baseUrl = 'https://basemaps.cartocdn.com/gl/{basemap}-gl-style/style.json';
+export const cartoBasemapsBaseUrl = 'https://basemaps.cartocdn.com/gl';
+
+const baseUrl = `${cartoBasemapsBaseUrl}/{basemap}-gl-style/style.json`;
+
+export const cartoStyleLayerGroups = [
```

Comment:
Add `CartoStyleLayerGroup` type? Then can extract `slug` and use in `applyCartoLayerGroupFilters` type defintion